chalicelib/checks/badge_checks.py

```diff
@@ -361,7 +361,12 @@ def repsets_have_bio_reps(connection, **kwargs):
             else:
                 audit_key = RELEASED_KEY
 
+            # check if single biological replicate
             if len(rep_dict.keys()) == 1:
+                many_replicates = '/static-sections/85520ecd-8df0-4d29-bd78-d8efa1f12ecf/'
```

Comment:
I wonder if it might be more easily maintained if you were to add a tag for this rather than relying on the static section.  Gives more flexibility to swap out to a similar but different static section.  It may be easier to remember to add the tag during submission and then eventually if this starts to appear more frequently set up a static header check to add the header based on a tag.  This is not blocking the merge and may not even be a good suggestion - just a thought I had.

Reply:
I  also like the idea of using a tag but am ok either way.

Reply:
That's a good idea, definitely more flexible.